blocks/card/card.css

```diff
@@ -0,0 +1,128 @@
+.card-box {
+  box-sizing: border-box;
+  border: 1px solid #dadada;
+  background-color: var(--color-background);
+  display: flex;
+  flex-direction: column;
+  border-radius: 1rem;
+  box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.1),
+    0px 1.4px 2.7px 0px rgba(0, 0, 0, 0.05), 0px 0px 1px 0px rgba(0, 0, 0, 0.16);
+}
+
+.card__body {
+  padding: 1.25rem 1rem;
+}
+
+.card__title {
+  color: var(--color-text-dark);
+  margin: 0;
+
+  @media (min-width: 48rem) {
+    font-size: var(--spectrum-title-size-m);
+  }
+
+  @media (min-width: 80rem) {
+    font-size: var(--spectrum-title-size-s);
+  }
+
+  @media (min-width: 105rem) {
+    font-size: var(--spectrum-title-size-xl);
+  }
+}
+
+.card__description {
+  color: var(--color-text);
+  margin: 0.25rem 0 0;
+
+  @media (min-width: 48rem) {
+    font-size: var(--spectrum-body-size-s);
+  }
+}
+
+.card__image {
+  border-radius: 1rem 1rem 0 0;
+}
+
+.card--vertical {
+  max-width: 22rem;
+  height: auto;
+}
+
+.card--vertical .card__description {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  /* line clamp to add ellipses to body paragraphs */
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+}
+
+/* Horizontal Card Styles */
+.card--horizontal {
+  max-width: 22rem;
+  height: auto;
+  border-radius: 0.5rem;
+
+  @media (min-width: 48rem) {
+    flex-direction: row-reverse;
```

Comment:
This functionality for switching between vertical and horizontal should also be using an `@container` query.